src/Analysis/Core/Impl/Interpreter/InterpreterConfiguration.cs

```diff
@@ -22,7 +22,7 @@ public sealed class InterpreterConfiguration : IEquatable<InterpreterConfigurati
         /// Constructs a new interpreter configuration based on the provided values.
         /// </summary>
         public InterpreterConfiguration(string interpreterPath = null, Version version = null) :
-            this(interpreterPath, string.Empty, null, null, default, version) { }
+            this(interpreterPath, pathVar: string.Empty, libPath: null, sitePackagesPath: null, architecture: default, version: version) { }
```

Comment:
I am deleting named argument, but there is one more benefit to named arguments. when a new parameter is added to a method, it reduces confusion whether right value is assigned to right parameter when literal constant is used as an argument.